packages/shared/src/components/post/PostActions.tsx

```diff
@@ -75,50 +87,89 @@ export function PostActions({
     return undefined;
   };
 
+  const toggleDownvote = () => {
+    if (!post) {
+      return;
+    }
+
+    if (!user) {
+      showLogin(AuthTriggers.Downvote);
+
+      return;
+    }
+
+    if (post.downvoted) {
+      cancelPostDownvote({ id: post.id });
+    } else {
+      downvotePost({ id: post.id });
+    }
+  };
+
   return (
-    <div className="flex justify-between py-2 px-4 rounded-16 border border-theme-divider-tertiary">
-      <QuaternaryButton
-        id="upvote-post-btn"
-        pressed={post.upvoted}
-        onClick={toggleUpvote}
-        icon={<UpvoteIcon secondary={post.upvoted} />}
-        aria-label="Upvote"
-        responsiveLabelClass={actionsClassName}
-        className="btn-tertiary-avocado"
-      >
-        Upvote
-      </QuaternaryButton>
-      <QuaternaryButton
-        id="comment-post-btn"
-        pressed={post.commented}
-        onClick={onComment}
-        icon={<CommentIcon secondary={post.commented} />}
-        aria-label="Comment"
-        responsiveLabelClass={actionsClassName}
-        className="btn-tertiary-blueCheese"
-      >
-        Comment
-      </QuaternaryButton>
-      <QuaternaryButton
-        id="bookmark-post-btn"
-        pressed={post.bookmarked}
-        onClick={onBookmark}
-        icon={<BookmarkIcon secondary={post.bookmarked} />}
-        aria-label="Bookmark"
-        responsiveLabelClass={actionsClassName}
-        className="btn-tertiary-bun"
-      >
-        Bookmark
-      </QuaternaryButton>
-      <QuaternaryButton
-        id="share-post-btn"
-        onClick={() => onShare(post)}
-        icon={<ShareIcon />}
-        responsiveLabelClass={actionsClassName}
-        className="btn-tertiary-cabbage"
+    <div className="flex items-center rounded-16 border border-theme-divider-tertiary">
+      <Card
+        className={classNames(
+          'flex !flex-row hover:border-theme-divider-tertiary gap-2',
+          {
+            'border-avocado-40 hover:!border-avocado-40 bg-avocado-50/8':
+              post.upvoted,
+            'border-ketchup-40 hover:!border-ketchup-40 bg-ketchup-50/8':
```

Comment:
For the value `-50`, we might need to introduce a new label for it. For the others, I think we can utilize some existing ones.
```suggestion
            'border-theme-color-avocado hover:!border-theme-color-avocado bg-avocado-50/8':
              post.upvoted,
            'border-theme-color-ketchup hover:!border-theme-color-ketchup bg-ketchup-50/8':
```

Reply:
@rebelchris from what I can see we don't have the `-50` in the theme as Lee said, can you check and propose how to name a label if needed?

Reply:
I just checked, and the value seems to be at `-40` still.

![image](https://github.com/dailydotdev/apps/assets/13744167/55df4a22-246c-4a39-9087-73bd90534903)

![Screenshot 2023-06-23 at 10 31 22 AM](https://github.com/dailydotdev/apps/assets/13744167/757b9476-69f5-4aca-a787-870522087c9e)

Reply:
Yeah it should just be color-ketchup as lee suggested, think Sketch is broken in attaching it to the right color for some reason.